tests/pipelines/kitctl/template.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-template
+  namespace: tekton-pipelines
+spec:
+  params:
+  - name: name
```

Comment:
Can we collapse these names?

Reply:
I have modified the data plane creation task to auto-detect the subnet and security group selector. Now we no longer need the name of the management cluster.